api/src/ContinuousPipe/Builder/GoogleContainerBuilder/HttpBuildCreator.php

```diff
@@ -56,7 +56,7 @@ public function startBuild(Artifact $sourceArtifact, string $gcbBuilderVersion):
                     ],
                     'steps' => [
                         [
-                            'name' => 'gcr.io/continuous-pipe-1042/cloud-builder:' . $gcbBuilderVersion,
+                            'name' => 'docker.io/continuouspipe/gcb-builder:' . $gcbBuilderVersion,
```

Comment:
Do we have the correct tags pushed for this?

Reply:
yes, `v8` as its defined in the Makefile

Reply:
https://hub.docker.com/r/continuouspipe/gcb-builder/tags/